tests/unit/test_library.py

```diff
@@ -246,7 +248,10 @@ def test_initialize_windows(self, mock_ctypes, mock_find_library, mock_open):
         lib = library.Library()
         lib.unload = mock.Mock()
 
-        mock_find_library.assert_called_once_with(library.Library.WINDOWS_JLINK_SDK_NAME)
+        if platform.architecture()[0] == '64bit':
```

Comment:
There should be a separate test for 64-bit where we mock and force the system to be 64-bit.  The existing tests should be mocked to treat the system as 32-bit.